### ISSUE TITLE
Hyperlink rich text editor's plugin does not work in multifield

### ISSUE THREAD

**Issue by boser87** [Adobe-Consulting-Services/acs-aem-commons#1241]
## Required Information

AEM Version, including Service Packs, Cumulative Fix Packs: 6.3.1.1
ACS AEM Commons Version: 3.8.4
Reproducible on Latest? Not tested

### Expected Behavior

My dialog contains a multifield with rich text editor (cq/gui/components/authoring/dialog/richtext) with fixed inline toolbar. One of the plugins of the rich text editor is the hyperlink plugin.

![image](https://user-images.githubusercontent.com/4405782/35632256-d35980e0-06ae-11e8-8d70-c387055c001a.png)

When I select a page from the path browser of the hyperlink plugin, the link should be added to the RTE and I should be able to submit the dialog.

### Actual Behavior

The actual behavior is that I cannot submit the dialog anymore (the V button becomes disabled).

![image](https://user-images.githubusercontent.com/4405782/35632366-27eedaec-06af-11e8-9d87-497ebc09b455.png)

### Steps to Reproduce

To reproduce the issue you can use the definition of the cq:dialog attached
[link plugin and rte inside acs commons multifield bug.txt](https://github.com/Adobe-Consulting-Services/acs-aem-commons/files/1682307/link.plugin.and.rte.inside.acs.commons.multifield.bug.txt)

Also attached a package with the full component that has this dialog

[acs_commons_multifiled_with_rte.zip](https://github.com/Adobe-Consulting-Services/acs-aem-commons/files/1682321/acs_commons_multifiled_with_rte.zip)

**Comment by davidjgonzalez** [comment]
@boser87 - are you using some acs commons feature here? It appears this is all OOTB UI widgetry, is that correct?

Assuming your not using anything from ACS commons for this, does his problem go away if you uninstall acs commmons?

Thanks!

**Comment by boser87** [comment]
Yes I am using ACS Commons feature. The multifield in which the rich text editor is included is the ACS Commons one.
